templates/vaultwarden-backup.xml

```diff
@@ -0,0 +1,33 @@
+  <Config Name="Environment File" Target="/.env" Default="/mnt/user/appdata/vaultwarden-backup/env" Mode="rw" Description="Environment File Location" Type="Path" Display="advanced" Required="false" Mask="false">/mnt/user/appdata/vaultwarden-backup/env</Config>
+  <Config Name="Rclone Remote Name" Target="RCLONE_REMOTE_NAME" Default="BitwardenBackup" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">BitwardenBackup</Config>
+  <Config Name="Rclone Remote Directory" Target="RCLONE_REMOTE_DIR" Default="/BitwardenBackup/" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">/BitwardenBackup/</Config>
+  <Config Name="Rclone Global Flag" Target="RCLONE_GLOBAL_FLAG" Default="" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false"></Config>
+  <Config Name="CRON" Target="CRON" Default="5 * * * *" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">5 * * * *</Config>
+  <Config Name="Enable Zip" Target="ZIP_ENABLE" Default="TRUE" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">TRUE</Config>
+  <Config Name="Zip Password" Target="ZIP_PASSWORD" Default="WHEREISMYPASSWORD" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">WHEREISMYPASSWORD</Config>
+  <Config Name="Zip Type" Target="ZIP_TYPE" Default="zip" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">zip</Config>
+  <Config Name="Backup File Suffix" Target="BACKUP_FILE_SUFFIX" Default="%Y%m%d" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">%Y%m%d</Config>
+  <Config Name="Days to Keep Backup" Target="BACKUP_KEEP_DAYS" Default="0" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">0</Config>
+  <Config Name="URL to Ping" Target="PING_URL" Default="" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false"></Config>
+  <Config Name="Enable SMTP" Target="MAIL_SMTP_ENABLE" Default="FALSE" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">FALSE</Config>
+  <Config Name="SMTP Variables" Target="MAIL_SMTP_VARIABLES" Default="" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false"></Config>
+  <Config Name="Mail To" Target="MAIL_TO" Default="" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false"></Config>
+  <Config Name="Mail When Successful" Target="MAIL_WHEN_SUCCESS" Default="TRUE" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">TRUE</Config>
+  <Config Name="Mail When Failed" Target="MAIL_WHEN_FAILURE" Default="TRUE" Mode="" Description="" Type="Variable" Display="advanced" Required="false" Mask="false">TRUE</Config>
```

Comment:
```suggestion
  <Config Name="Send email after failure" Target="MAIL_WHEN_FAILURE" Default="FALSE" Mode="" Description="Whether to send emails after a failed backup" Type="Variable" Display="advanced" Required="false" Mask="false">FALSE</Config>
```

Since STMP is off by default, this should also be off by default.